styles.css

```diff
@@ -0,0 +1,95 @@
+body.index{
+    background-color: #F0F8FF;
+}
+
+body.about{
```

Comment:
very small nitpick: ensure that spacing before curly brackets is consistent for all blocks